### DISCUSSION TITLE
Issue on FB Login

### DISCUSSION THREAD

**Discussion by harshitgoyalcb** [cuongdevjs/reactjs-social-login#105] [Q&A] [answered]
Hi there,

I have tried implementing Fb social login using reactjs-social-login npm package but after authenticating login credentials i got this issue

and got nothing in response in which we usually get acces-token and user-data

![image](https://user-images.githubusercontent.com/126859872/233945568-8bd150d7-4a47-4b48-8d1d-4df967206bc4.png)

![image](https://user-images.githubusercontent.com/126859872/233945716-61180633-49d1-4ed7-bebd-e1b1b2c1ca5e.png)

And I also tried with https requests still error is same

could anyone explain why this is happening

**Comment by cuongdevjs** [comment, marked as answer]
I think it is new to Facebook. I can use it locally before. But seems now they don't allow to use it in an HTTP environment. 
If you use HTTPS and still got errors, I think the problem is your application configuration.
